mathesar_ui/src/api/utils/requestUtils.ts

```diff
@@ -166,6 +166,13 @@ export function deleteAPI<T>(url: string): CancellablePromise<T> {
   return sendXHRRequest('DELETE', url);
 }
 
+export function deletebulkAPI<T>(
+  url: string,
+  data: unknown,
+): CancellablePromise<T> {
+  return sendXHRRequest('DELETE', url, data);
+}
+
```

Comment:
Please update the existing deleteAPI function to accept an optional data argument instead of the new function.